### PR TITLE
[UI-92] Matching Mocks for Workstation Alert Windows

### PR DESCRIPTION
**Pull request by Shadae**
screen shot 2018-10-16 at 11 17 43 am

* undefined because I'm pushing it to show without data locally so I can style it.
New Screen Shot with Download Button
screen shot 2018-10-16 at 5 19 42 pm

### CODE REVIEW

File: src/update_available_dialog.js [hunk 1]
```diff
@@ -10,13 +10,14 @@ function open(updateInfo, workstationVersion) {
...
       alwaysOnTop: true,
-      width: 450,
+      width: 433,
       height: 145,
       resizable: false,
       minimizable: false,
       maximizable: false,
       alwaysOnTop: true
     });
+    updateAvailableDialog.openDevTools({mode:'undocked'});
```
Comment (Shadae) [inline src/update_available_dialog.js]: oops left this in!

File: assets/css/main.css [hunk 2]
```diff
@@ -90,3 +105,50 @@ div.col-3 {
   font-size: 10px;
   font-weight: 400;
 }
+
+#logo-alert {
+  content: url('../images/chef-ws-icon-black.png');
```
Comment (jonsmorrow) [inline assets/css/main.css]: should we use the svg here like on the about page?

File: assets/css/main.css [hunk 3]
```diff
@@ -29,14 +29,29 @@ section, p {
...
 
+button.primary {
+  height: 20px;
+  width: 72px;
+  font-size: 13px;
+  color: white;
+  border: none;
+  font-weight: 300;
+  border-radius: 3.5px;
+  background-image: linear-gradient(to bottom, #6cb3fa 10%, #067dff 100%);
```
Comment (jonsmorrow) [inline assets/css/main.css]: Also non-blocking but I have concern about making this blue. I wonder if there is a way to get the system styling for this? If not and maybe until we figure that out lets go with it.

File: src/no_update.html [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
+      <div id="logo-alert"></div>
+      <div id="text-block">
+        <div class="title large">
+          Already up-to-date!
         </div>
-        <div>
-          <script>document.write(workstation.getVersion())</script> is the latest
-          <script>document.write(workstation.getUpdateChannel())</script> release available.
+        <div class="message">
+          Chef Workstation <script>document.write(getAppVersion())</script> is the latest <br>
```
Comment (jonsmorrow) [inline src/no_update.html]: <script>document.write(getDisplayName())</script>
<script>document.write(workstation.getVersion())</script>
<script>document.write(workstation.getUpdateChannel())</script>

File: src/update_available.html [hunk 5]
```diff
@@ -10,29 +10,31 @@
...
-          </script>
-          is the latest
-          <script>document.write(workstation.getUpdateChannel())</script>
-          release, you have
-          <script>document.write(workstation.getVersion())</script>.
+        <div class="message">
+          <!-- getDisplayName() = Chef Workstation -->
+          <script>document.write(getDisplayName())</script> 
+          <!-- getReleaseChannel = new release -->
+          <script>document.write(getReleaseChannel())</script>
```
Comment (jonsmorrow) [inline src/update_available.html]: <script>document.write(workstation.getUpdateChannel())</script>

File: src/update_available.html [hunk 6]
```diff
@@ -10,29 +10,31 @@
...
+          is now available. You currently have
+          <script>document.write(currentWindow.workstationVersion)</script>
```
Comment (jonsmorrow) [inline src/update_available.html]: <script>document.write(workstation.getUpdateChannel())</script>

**Pull request closed without merging**